probe-dictionary/environment.json

```diff
@@ -3364,5 +3415,1229 @@
+          "optout": true,
+          "revisions": {
+            "firstVersion": "42",
+            "last": "latest"
+          }
+        }
+      ]
+    },
+    "name": "system.gfx.D2DEnabled",
+    "type": "environment"
+  },
+  "environment/system.gfx.DWriteEnabled": {
+    "history": {
+      "release": [
+        {
+          "description": "Whether or not DirectWrite is enabled. This is Windows only.",
+          "bug_numbers": [1122052],
+          "details": {
+            "kind": "boolean"
+          },
+          "expiry_version": "never",
+          "optout": true,
+          "revisions": {
+            "firstVersion": "42",
+            "last": "latest"
+          }
+        }
+      ],
+      "beta": [
+        {
+          "description": "Whether or not DirectWrite is enabled. This is Windows only.",
+          "bug_numbers": [1122052],
+          "details": {
+            "kind": "boolean"
+          },
+          "expiry_version": "never",
+          "optout": true,
+          "revisions": {
+            "firstVersion": "42",
+            "last": "latest"
+          }
+        }
+      ],
+      "nightly": [
+        {
+          "description": "Whether or not DirectWrite is enabled. This is Windows only.",
+          "bug_numbers": [1122052],
+          "details": {
+            "kind": "boolean"
+          },
+          "expiry_version": "never",
+          "optout": true,
+          "revisions": {
+            "firstVersion": "42",
+            "last": "latest"
+          }
+        }
+      ]
+    },
+    "name": "system.gfx.DWriteEnabled",
+    "type": "environment"
+  },
+  "environment/system.gfx.ContentBackend": {
+    "history": {
+      "release": [
+        {
+          "description": "The name of the content backend.",
+          "bug_numbers": [1302240],
+          "details": {
+            "kind": "string"
+          },
+          "expiry_version": "never",
+          "optout": true,
+          "revisions": {
+            "firstVersion": "51",
+            "last": "latest"
+          }
+        }
+      ],
+      "beta": [
+        {
+          "description": "The name of the content backend.",
+          "bug_numbers": [1302240],
+          "details": {
+            "kind": "string"
+          },
+          "expiry_version": "never",
+          "optout": true,
+          "revisions": {
+            "firstVersion": "51",
+            "last": "latest"
+          }
+        }
+      ],
+      "nightly": [
+        {
+          "description": "The name of the content backend.",
+          "bug_numbers": [1302240],
+          "details": {
+            "kind": "string"
+          },
+          "expiry_version": "never",
+          "optout": true,
+          "revisions": {
+            "firstVersion": "51",
+            "last": "latest"
+          }
+        }
+      ]
+    },
+    "name": "system.gfx.ContentBackend",
+    "type": "environment"
+  },
+  "environment/system.gfx.adapters.description": {
```

Comment:
I agree, for the arrays and dictionaries we should have something that highlights the structure.
How about something like `environment/system.gfx.adapters[<index>].description`?

For now this is only for display purposes.

Reply:
Works for me! I had to remove the `<>` from the name otherwise it wouldn't render correctly (unless we encoded the "<" in the JSON file, which seemed ugly). I've opted for `environment/system.gfx.adapters[index].description`.